portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/DMSearch.testcase

```diff
@@ -49,12 +49,27 @@ definition {
 	}
 
 	@description = "This is a test for LPS-98063. It checks that a user can search for document by synonyms."
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanSearchForDocumentBySynonyms {
+		property test.run.environment = "EE";
```

Comment:
Hey @jaydawu @liuxingye should this only be run on DXP?

Reply:
@austinchiang yeah, this feature works on DXP only.